zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -345,10 +345,10 @@ public static UserTaskItem toUserTask(final UserTaskEntity t) {
         .processDefinitionKey(t.processDefinitionId())
         .elementInstanceKey(t.flowNodeInstanceId())
         .processDefinitionId(t.bpmnProcessId())
-        .state(t.state())
+        .state(UserTaskItem.StateEnum.fromValue(String.valueOf(t.state())))
```

Comment:
🔧 I haven't looked into this in my IDE but it feels like this could be done in a simpler way. What does `.state()` expect as a value and what is the type of `t.state()`?

Reply:
Yes it possible to make it simpler, this expects receive a UserItem.StateEnum

I have updated the code on this way now: `.state(UserTaskItem.StateEnum.fromValue(t.state().name()))`

Reply:
The type of `t.state()` is a different enum class, I assume?

Reply:
Yes ;) It is from `UserItem`, now its like in the same way as the others.

Reply:
OK, I figured that, yeah. Good enough for me 👍